GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAndroidResource.java

```diff
@@ -313,6 +313,23 @@ public synchronized final InputStream getStream() throws IOException{
         return stream;
     }
 
+    /**
```

Comment:
Yes, it has a CONSTUCTOR that accepts an input stream. Problem is, the asset loader constructs the resource in one place and opens it in another. The method is protected so that only subclasses, like the one Srini is trying to write, can provide an open stream.